package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "gaia-button",
+  "version": "0.0.5",
+  "homepage": "https://github.com/gaia-components/gaia-button",
+  "license": "MIT",
+  "main": "gaia-button.js",
+  "devDependencies": {
```

Comment:
I don't mean publish to npm, just make `npm install gaia-components/gaia-button` possible.